UpdatePaycoinNode.sh

```diff
@@ -1,16 +1,47 @@
 #!/bin/bash
+version=$(curl -s https://api.github.com/repos/PaycoinFoundation/Paycoin/releases/latest | grep 'tag_' | cut -d\" -f4 | sed 's/\v//g')
+latest=$(curl -s https://api.github.com/repos/PaycoinFoundation/Paycoin/releases/latest | grep 'browser_' | cut -d\" -f4 | grep 'linux64.zip')
 echo "### Changing to paycoind directory"
 cd ~
-echo "### Stopping Paycoin Server"
-./paycoind stop
-echo "### Changing to home directory"
-cd ~
-echo "### Downloading Paycoin Core 0.3.3.0"
-wget https://github.com/PaycoinFoundation/paycoin/releases/download/v0.3.3.0/linux64.zip
-echo "### Installing Paycoin Core 0.3.3.0"
+xpy_version=$(./paycoind getinfo | grep '"version"' | cut -d\" -f4 | sed 's/\v//g' )
+# Check we are running the same version; otherwise; update
+
+if [[ "${xpy_version}" == "" ]]
+then
+    echo "### Paycoin server not running, Unable to check version"
+    echo "### The latest is available version is v${version}"
```

Comment:
Again funny sentence structure here. It's got one too many 'is', something like 'The latest available version is v${version}'  works better.